apps/hip/models.py

```diff
@@ -105,6 +105,10 @@ class StreamAndNavHeadingBlock(blocks.StructBlock):
 class StaticPage(Page):
     """A Page with only sections of static content."""
 
+    max_count = 1
```

Comment:
If we're keeping this as the `StaticPage`, should we remove the `max_count` and `parent_page_types` as well?